lighthouse-core/report/html/report-styles.css

```diff
@@ -387,14 +386,20 @@
   margin-right: 0;
 }
 
-.lh-audit__title {
+.lh-audit__display-text {
   flex: 1;
 }
 
+/* Prepend display text with separator. But not in Opportunities. */
+.lh-audit-group:not(.lh-audit-group--load-opportunities) .lh-audit__display-text:not(:empty):before {
+  content: '—';
```

Comment:
```suggestion
  content: '\2014';
```

Reply:
can this just go in the HTML? Kind of weird to do it with a pseudo-element

Reply:
why? dt encoding issues are resolved

Reply:
@brendankenny can't just be HTML, need some sort of logic check. makes sense to me to have the display + logic in one place here.

Reply:
> can't just be HTML, need some sort of logic check. makes sense to me to have the display + logic in one place here.

I don't think it needs to be. Opportunities and the rest of the audits share text with a `.lh-audit__display-text` class, but they actually come from different templates, `#tmpl-lh-opportunity` vs `#tmpl-lh-audit`. If the `#tmpl-lh-audit` template gets the em dash inline, then it'll only show up there

Reply:
Good catch on the template thing. But would still need a length check - and can't make a CSS rule for preceding sibling (pseudo code below).

![image](https://user-images.githubusercontent.com/4071474/57004725-05e20900-6b86-11e9-819b-9c213ec01542.png)

```
.lh-audit__separator {
  margin: 0 var(--audit-item-gap);
}
/* not actually a thing */
.lh-audit__display-text:not(:empty) - .lh-audit__separator {
  display: none;
}
```